swingset-demo/src/main/java/com/nqadmin/swingset/demo/Example4UsingHelper.java

```diff
@@ -202,6 +197,10 @@ protected void addNewRecordToCmbNavigator() {
 
 	@Override
 	protected void bindComponents() throws Exception {
+		// SET COMBO OPTIONS
+		//cmbPartColor.setOptions(new String[] { "Red", "Green", "Blue" });
+		cmbPartColor.setOptions(Arrays.asList(new String[] { "Red", "Green", "Blue" }));
+		
```

Comment:
Curiosity. Does moving this make a difference or avoid an issue?

Reply:
Avoids an issue. In my helper bindComponents is called before addComponents. This results in the first record showing SSComboBoxes as blank. If you navigate next/back then it refreshes.

I may add an explicit abstract method like populateSSCombos() to the form helper to guide the developer and call that before bindComponents() or addComponents().